packages/hydrogen/src/product/VariantSelector.ts

```diff
@@ -47,6 +48,8 @@ type VariantSelectorProps = {
   productPath?: string;
   /** Should the VariantSelector wait to update until after the browser navigates to a variant. */
   waitForNavigation?: boolean;
+  /** An optional selected variant to use for the initial state */
+  selectedVariant?: Maybe<PartialDeep<ProductVariant>>;
```

Comment:
I keep bouncing around on the naming. Any preference here @wizardlyhel?

`defaultOptions`, `defaultSelectedVariant`, `defaultSelectedOptions`

Reply:
keep it `selectedVariant` as it is what we generated in skeleton template as part of product query